sample-application/device-assistant/pyproject.toml

```
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "device-assistant"
+version = "0.1.0"
+description = ""
+authors = ["Observability workshop <ikulakowska@microsoft.com>"]
```

```suggestion
authors = []
```

I think it should work with an empty list of authors